.github/ci-scripts/job_runner.py

```diff
@@ -0,0 +1,109 @@
+import argparse
```

Comment:
Cool and if you add this we could also probably just run this independently

```
/// script
/// dependencies ...
/// script
```

Reply:
Updated.